postgresql_lo_streamer.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "rails", "~> 4.0"
+  s.add_dependency "rails", "~> 5.0"
```

Comment:
if there is nothing preventing the gem from working in a 4.0 install let's change to `>= 4.0`